### ISSUE TITLE
While running python 3.6 - cannot import name 'PGDialect_psycopg2'

### ISSUE THREAD

**Issue by coletivo** [jkehler/awslambda-psycopg2#27]
Hello there - 
I am running python 3.6 with sqlalchemy and using this repo as well. I copied over the `psycopg2-3.6` directory and renamed it `psycopg2`.

I am then zipping the file and uploading it to Lambda. However, when I try to run it, I get the following error:
`cannot import name 'PGDialect_psycopg2'`

Do you know what is happening here?

**Comment by vnktsh** [comment]
I'm too getting an issue with python 3.6. 

Lambda env: Python 3.6
aws-psycopg2: psycpog2-3.6 from this repo (renamed in zip file before uploading to lambda)

Error:

```
/var/task/psycopg2/_psycopg.so: undefined symbol: PyUnicodeUCS4_DecodeUTF8: ImportError
Traceback (most recent call last):
File "/var/task/main.py", line 159, in lambda_handler
init_sqlalchemy(db_connect_url)
File "/var/task/main.py", line 74, in init_sqlalchemy
engine = create_engine(dbname, echo=False)
File "/var/task/sqlalchemy/engine/__init__.py", line 419, in create_engine
return strategy.create(*args, **kwargs)
File "/var/task/sqlalchemy/engine/strategies.py", line 80, in create
dbapi = dialect_cls.dbapi(**dbapi_args)
File "/var/task/sqlalchemy/dialects/postgresql/psycopg2.py", line 584, in dbapi
import psycopg2
File "/var/task/psycopg2/__init__.py", line 50, in <module>
from psycopg2._psycopg import ( # noqa
ImportError: /var/task/psycopg2/_psycopg.so: undefined symbol: PyUnicodeUCS4_DecodeUTF8
```

Then tried compiling myself a new lib:
PostgresSQL source code: 10.1
Psycopg2 source code: 2.7.4

**same error**

**Comment by jkehler** [comment]
@vnktsh @coletivo Did either of you find a solution to this?

**Comment by coletivo** [comment]
As I was using lambda, just used the python 2.7 version and it worked

**Comment by vnktsh** [comment]
@jkehler Hi Jeff, It worked for me finally. I have compiled my own psycopg2 from the sources for Py 3.6 and used it. I haven't figured out why the one on this repo didn't work, but mine worked.

**Comment by jkehler** [comment]
@coletivo @vnktsh I finally got some free time to play around with this today and I was not able to find any problem with the included 3.6 library. The only way I was able to replicate the PyUnicodeUCS4_DecodeUTF8 error was if I used the python 2.7 library on a lambda 3.6 instance. There are 2 folders in this directory and if you're using python 3.6 lambda environment you need to use the psycopg2-3.6 folder and rename it by removing the -3.6 part.
